src/tribol/interface/simple_tribol.cpp

```diff
@@ -31,20 +31,11 @@ namespace numerics = axom::numerics;
 // free functions for simple API usage
 //------------------------------------------------------------------------------
 
-int Initialize(const int dim, bool init_slic)
+int Initialize(const int dim, bool TRIBOL_UNUSED_PARAM(init_slic))
 {
-   // initialize slic
-   if(init_slic)
-   {
-      axom::slic::finalize();
-      axom::slic::initialize();
-      std::string format = "[<LEVEL>]: <MESSAGE> \n";
-      axom::slic::setLoggingMsgLevel( axom::slic::message::Info );
-
-      axom::slic::addStreamToAllMsgLevels(
-         new axom::slic::GenericOutputStream( &std::cout,format) );
-   }
-
+   // Note, slic initialization moved into tribol::initialize(). 
+   // Function prototype not modified due to shroud layer
```

Comment:
👍

Reply:
If we no longer need the `init_slic` parameter, we can (and should) change the shroud interface in a follow-up PR.

Reply:
I removed these comments because we actually don't have slic initialization in tribol::initialize() per design discussion.